src/AltinnCore/Designer/Controllers/ApplicationMetadataController.cs

```diff
@@ -85,7 +85,10 @@ public ActionResult CreateApplicationMetadata(string org, string app)
             }
             else
             {
-                _repository.CreateApplication(org, app);
+                // At the moment we send empty string as application title until we support multiple
+                // languages (it will now get set to the "app" value in the CreateApplication method
+                // if title is undefined).
+                _repository.CreateApplication(org, app, string.Empty);
```

Comment:
Why do you not use app instead of string.Empty. An app with blank title does not give any sence.

Reply:
Done.